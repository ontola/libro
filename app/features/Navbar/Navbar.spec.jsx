import { BlankNode, Literal, NamedNode } from 'rdflib';
import React from 'react';

import { NS } from '../../helpers/LinkedRenderStore';

import components, { Component } from './Navbar';

const resource = new NamedNode('https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1');
const totalCount = 378;
const unreadCount = 6;

const resources = {
  [resource]: {
    [NS.rdf('type')]: [
      new NamedNode('http://www.w3.org/ns/iana/media-types/text/n3#Resource'),
      new NamedNode('http://www.w3.org/2007/ont/link#Document'),
      new NamedNode('http://www.w3.org/2007/ont/link#RDFDocument'),
    ],
    [NS.argu('contains')]: NS.app('nederland'),
  },
  [NS.app('nederland')]: {
    [NS.schema('description')]: new Literal(''),
    [NS.argu('blogPosts')]: NS.app('nederland/blog'),
    [NS.schema('image')]: NS.app('media_objects/17843'),
    [NS.schema('dateModified')]: Literal.fromDate(new Date('2017-10-20T14:04:15+02:00')),
    [NS.schema('dateCreated')]: Literal.fromDate(new Date('2017-10-20T14:04:15+02:00')),
    [NS.dc('identifier')]: NS.app('nederland'),
    [NS.rdf('type')]: NS.argu('Page'),
    [NS.argu('navigationsMenu')]: NS.app('nederland/menus/navigations'),
    [NS.argu('voteMatches')]: NS.app('nederland/vote_matches'),
    [NS.argu('questions')]: NS.app('nederland/discussions'),
    [NS.schema('potentialAction')]: NS.app('nederland/vote_matches/new'),
    [NS.schema('name')]: new Literal('Nederland'),
  },
  [NS.app('media_objects/17843')]: {
    [NS.ll('nop')]: Literal.fromNumber(0),
    [NS.schema('isPartOf')]: NS.app('nederland'),
    [NS.schema('url')]: NS.app('photos/17843/nederland.png'),
    [NS.schema('dateCreated')]: Literal.fromDate(new Date('2016-06-01T16:36:38+02:00')),
    [NS.dc('identifier')]: NS.app('media_objects/17843'),
    [NS.rdf('type')]: NS.schema('ImageObject'),
    [NS.schema('contentUrl')]: new NamedNode('https://argu-logos.s3.amazonaws.com/photos/734/freeformatter-decoded.png'),
    [NS.schema('thumbnail')]: new NamedNode('https://argu-logos.s3.amazonaws.com/photos/734/icon_freeformatter-decoded.png'),
  },
  [NS.app('nederland/menus/navigations')]: {
    [NS.schema('isPartOf')]: NS.app('nederland/menus'),
    [NS.argu('menuItems')]: new BlankNode('12_g69837729252540'),
    [NS.schema('name')]: new Literal('Navigations'),
    [NS.dc('identifier')]: NS.app('nederland/menus/navigations'),
    [NS.rdf('type')]: NS.argu('MenuItem'),
  },
  [new BlankNode('12_g69837729252540')]: {
    [NS.rdf('_1')]: NS.app('nederland/menus/navigations#forums'),
    [NS.rdf('_0')]: NS.app('nederland/menus/navigations#settings'),
    [NS.dc('identifier')]: new BlankNode('12_g69837729252540'),
    [NS.rdf('type')]: NS.rdf('Seq'),
  },
  [NS.app('nederland/menus/navigations#settings')]: {
    [NS.schema('isPartOf')]: NS.app('nederland/menus/navigations'),
    [NS.schema('image')]: new NamedNode('http://fontawesome.io/icon/gear'),
    [NS.schema('name')]: new Literal('Instellingen'),
    [NS.dc('identifier')]: NS.app('nederland/menus/navigations#settings'),
    [NS.argu('href')]: new Literal('https://argu.localdev/nederland/settings'),
    [NS.rdf('type')]: NS.argu('MenuItem'),
  },
  [NS.app('nederland/menus/navigations#forums')]: {
    [NS.schema('isPartOf')]: NS.app('nederland/menus/navigations'),
    [NS.argu('menuItems')]: new BlankNode('12_g69837780660080'),
    [NS.schema('name')]: new Literal('Forum'),
    [NS.dc('identifier')]: NS.app('nederland/menus/navigations#forums'),
    [NS.rdf('type')]: NS.argu('MenuSection'),
  },
  [new BlankNode('12_g69837780660080')]: {
    [NS.rdf('_1')]: NS.app('nederland/menus/navigations#forums.new_discussion'),
    [NS.rdf('_2')]: NS.app('nederland/menus/navigations#forums.activity'),
    [NS.rdf('_0')]: NS.app('nederland/menus/navigations#forums.overview'),
    [NS.dc('identifier')]: new BlankNode('12_g69837780660080'),
    [NS.rdf('type')]: NS.rdf('Seq'),
  },
  [NS.app('nederland/menus/navigations#forums.overview')]: {
    [NS.schema('isPartOf')]: NS.app('nederland/menus/navigations#forums'),
    [NS.schema('image')]: new NamedNode('http://fontawesome.io/icon/th-large'),
    [NS.schema('name')]: new Literal('Overzicht'),
    [NS.dc('identifier')]: NS.app('nederland/menus/navigations#forums.overview'),
    [NS.argu('href')]: NS.app('nederland/forum'),
    [NS.rdf('type')]: NS.argu('MenuItem'),
  },
  [NS.app('nederland/menus/navigations#forums.new_discussion')]: {
    [NS.schema('isPartOf')]: NS.app('nederland/menus/navigations#forums'),
    [NS.schema('image')]: new NamedNode('http://fontawesome.io/icon/plus'),
    [NS.schema('name')]: new Literal('Nieuwe discussie'),
    [NS.dc('identifier')]: NS.app('nederland/menus/navigations#forums.new_discussion'),
    [NS.argu('href')]: NS.app('nederland/forum/discussions/new'),
    [NS.rdf('type')]: NS.argu('MenuItem'),
  },
  [NS.app('nederland/menus/navigations#forums.activity')]: {
    [NS.schema('isPartOf')]: NS.app('nederland/menus/navigations#forums'),
    [NS.schema('image')]: new NamedNode('http://fontawesome.io/icon/feed'),
    [NS.schema('name')]: new Literal('Activiteit'),
    [NS.dc('identifier')]: NS.app('nederland/menus/navigations#forums.activity'),
    [NS.argu('href')]: NS.app('nederland/forum/feed'),
    [NS.rdf('type')]: NS.argu('MenuItem'),
  },
  [NS.app('n')]: {
    [NS.ll('nop')]: Literal.fromNumber(0),
    [NS.as('name')]: new Literal('Notifications'),
    [NS.argu('defaultType')]: new Literal('infinite'),
    [NS.dc('identifier')]: NS.app('n'),
    [NS.rdf('type')]: [
      NS.argu('Collection'),
      NS.as('Collection'),
    ],
    [NS.schema('potentialAction')]: NS.app('n/new'),
    [NS.as('totalItems')]: Literal.fromNumber(totalCount),
    [NS.argu('unreadCount')]: Literal.fromNumber(unreadCount),
    [NS.argu('iriTemplate')]: new Literal('https://argu.localdev/n{?filter%5B%5D,page,page_size,type,before,sort%5B%5D}'),
    [NS.as('pages')]: NS.app('n?type=infinite&before=2018-03-22%2010%3A42%3A42'),
    [NS.argu('createAction')]: NS.app('n/new'),
  },
};

describeView('Navbar', components, resources, resource, () => {
  set('ch', () => <Component />);

  describe('as default', () => {
    it('renders the organization name', () => {
      expect(subject.find('OrganizationName .OrganizationName__value'))
        .toHaveText('Nederland');
    });

    it('renders the forum items', () => {
      const forumSection = subject.find('.NavBarContent__top MenuSection');
      expect(forumSection.find('MenuItemSidebar[data-test="Seq-0-https://argu.dev/nederland/menus/navigations#forums.overview"] SideBarLinkLink Link'))
        .toHaveText('Overzicht');
      expect(forumSection.find('MenuItemSidebar[data-test="Seq-1-https://argu.dev/nederland/menus/navigations#forums.new_discussion"] SideBarLinkLink Link'))
        .toHaveText('Nieuwe discussie');
      expect(forumSection.find('MenuItemSidebar[data-test="Seq-2-https://argu.dev/nederland/menus/navigations#forums.activity"] SideBarLinkLink Link'))
        .toHaveText('Activiteit');
    });
  });
});
