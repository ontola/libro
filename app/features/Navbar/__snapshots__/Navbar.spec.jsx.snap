// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar as default renders the correct representation 1`] = `
<NavBarContent
  orgColor="rgb(71, 86, 104)"
>
  <div
    className="NavBarContent NavBarContent--white-text"
    style={
      Object {
        "backgroundColor": "rgb(71, 86, 104)",
        "color": "white",
        "flex": "1",
      }
    }
  >
    <div
      className="NavBarContent__top"
    >
      <Connect(LinkedResourceContainer)
        forceRender={true}
        subject={
          NamedNode {
            "sI": 99,
            "term": "find",
            "termType": "NamedNode",
            "value": "https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1",
          }
        }
        topology={
          NamedNode {
            "sI": 45,
            "term": "sidebarBlock",
            "termType": "NamedNode",
            "value": "https://argu.co/ns/core#sidebarBlock",
          }
        }
      >
        <LinkedResourceContainer
          forceRender={true}
          loadLinkedObject={[Function]}
          reloadLinkedObject={[Function]}
          subject={
            NamedNode {
              "sI": 99,
              "term": "find",
              "termType": "NamedNode",
              "value": "https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1",
            }
          }
          topology={
            NamedNode {
              "sI": 45,
              "term": "sidebarBlock",
              "termType": "NamedNode",
              "value": "https://argu.co/ns/core#sidebarBlock",
            }
          }
          version="test0"
        >
          <div
            className="view-overridden"
            style={
              Object {
                "display": "inherit",
              }
            }
          >
            <linkedSubject[ConnectedProp]
              forceRender={true}
              label={
                NamedNode {
                  "sI": 74,
                  "term": "contains",
                  "termType": "NamedNode",
                  "value": "https://argu.co/ns/core#contains",
                }
              }
            >
              <ConnectedProp
                forceRender={true}
                label={
                  NamedNode {
                    "sI": 74,
                    "term": "contains",
                    "termType": "NamedNode",
                    "value": "https://argu.co/ns/core#contains",
                  }
                }
                subject={
                  NamedNode {
                    "sI": 99,
                    "term": "find",
                    "termType": "NamedNode",
                    "value": "https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1",
                  }
                }
              >
                <Property
                  dispatch={[Function]}
                  forceRender={true}
                  label={
                    NamedNode {
                      "sI": 74,
                      "term": "contains",
                      "termType": "NamedNode",
                      "value": "https://argu.co/ns/core#contains",
                    }
                  }
                  limit={1}
                  subject={
                    NamedNode {
                      "sI": 99,
                      "term": "find",
                      "termType": "NamedNode",
                      "value": "https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1",
                    }
                  }
                  version="test0"
                >
                  <linkedSubject[Connect(Contains)]
                    dispatch={[Function]}
                    forceRender={true}
                    label={
                      NamedNode {
                        "sI": 74,
                        "term": "contains",
                        "termType": "NamedNode",
                        "value": "https://argu.co/ns/core#contains",
                      }
                    }
                    limit={1}
                    linkedProp={
                      NamedNode {
                        "sI": 69,
                        "term": "1",
                        "termType": "NamedNode",
                        "value": "https://argu.dev/o/1",
                      }
                    }
                    subject={
                      NamedNode {
                        "sI": 99,
                        "term": "find",
                        "termType": "NamedNode",
                        "value": "https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1",
                      }
                    }
                    version="test0"
                  >
                    <Connect(Contains)
                      dispatch={[Function]}
                      forceRender={true}
                      label={
                        NamedNode {
                          "sI": 74,
                          "term": "contains",
                          "termType": "NamedNode",
                          "value": "https://argu.co/ns/core#contains",
                        }
                      }
                      limit={1}
                      linkedProp={
                        NamedNode {
                          "sI": 69,
                          "term": "1",
                          "termType": "NamedNode",
                          "value": "https://argu.dev/o/1",
                        }
                      }
                      subject={
                        NamedNode {
                          "sI": 99,
                          "term": "find",
                          "termType": "NamedNode",
                          "value": "https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1",
                        }
                      }
                      version="test0"
                    >
                      <Contains
                        dispatch={[Function]}
                        forceRender={true}
                        label={
                          NamedNode {
                            "sI": 74,
                            "term": "contains",
                            "termType": "NamedNode",
                            "value": "https://argu.co/ns/core#contains",
                          }
                        }
                        limit={1}
                        linkedProp={
                          NamedNode {
                            "sI": 69,
                            "term": "1",
                            "termType": "NamedNode",
                            "value": "https://argu.dev/o/1",
                          }
                        }
                        subject={
                          NamedNode {
                            "sI": 99,
                            "term": "find",
                            "termType": "NamedNode",
                            "value": "https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1",
                          }
                        }
                        version="test0"
                      >
                        <Connect(LinkedResourceContainer)
                          forceRender={true}
                          subject={
                            NamedNode {
                              "sI": 69,
                              "term": "1",
                              "termType": "NamedNode",
                              "value": "https://argu.dev/o/1",
                            }
                          }
                        >
                          <LinkedResourceContainer
                            forceRender={true}
                            loadLinkedObject={[Function]}
                            reloadLinkedObject={[Function]}
                            subject={
                              NamedNode {
                                "sI": 69,
                                "term": "1",
                                "termType": "NamedNode",
                                "value": "https://argu.dev/o/1",
                              }
                            }
                            version="test1"
                          >
                            <div
                              className="view-overridden"
                              style={
                                Object {
                                  "display": "inherit",
                                }
                              }
                            >
                              <div
                                className="NavBarContent__switcher"
                              >
                                <Connect(SideBarCollapsible)
                                  id="<https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1>-menu-items"
                                  labelComp={
                                    <linkedSubject[LDLink]>
                                      <linkedSubject[ConnectedProp]
                                        label={
                                          NamedNode {
                                            "sI": 59,
                                            "term": "name",
                                            "termType": "NamedNode",
                                            "value": "http://schema.org/name",
                                          }
                                        }
                                      />
                                    </linkedSubject[LDLink]>
                                  }
                                >
                                  <SideBarCollapsible
                                    id="<https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1>-menu-items"
                                    labelComp={
                                      <linkedSubject[LDLink]>
                                        <linkedSubject[ConnectedProp]
                                          label={
                                            NamedNode {
                                              "sI": 59,
                                              "term": "name",
                                              "termType": "NamedNode",
                                              "value": "http://schema.org/name",
                                            }
                                          }
                                        />
                                      </linkedSubject[LDLink]>
                                    }
                                    onClickToggle={[Function]}
                                    open={false}
                                  >
                                    <div
                                      className="SideBarCollapsible"
                                      data-test="SideBarCollapsible-sideBarCollapsible"
                                    >
                                      <div
                                        className="SideBarCollapsible__label"
                                        data-test="SideBarCollapsible-label"
                                      >
                                        <linkedSubject[LDLink]>
                                          <LDLink
                                            subject={
                                              NamedNode {
                                                "sI": 69,
                                                "term": "1",
                                                "termType": "NamedNode",
                                                "value": "https://argu.dev/o/1",
                                              }
                                            }
                                          >
                                            <Link
                                              replace={false}
                                              to="/o/1"
                                            >
                                              <a
                                                href="/o/1"
                                                onClick={[Function]}
                                              >
                                                <linkedSubject[ConnectedProp]
                                                  label={
                                                    NamedNode {
                                                      "sI": 59,
                                                      "term": "name",
                                                      "termType": "NamedNode",
                                                      "value": "http://schema.org/name",
                                                    }
                                                  }
                                                >
                                                  <ConnectedProp
                                                    label={
                                                      NamedNode {
                                                        "sI": 59,
                                                        "term": "name",
                                                        "termType": "NamedNode",
                                                        "value": "http://schema.org/name",
                                                      }
                                                    }
                                                    subject={
                                                      NamedNode {
                                                        "sI": 69,
                                                        "term": "1",
                                                        "termType": "NamedNode",
                                                        "value": "https://argu.dev/o/1",
                                                      }
                                                    }
                                                  >
                                                    <Property
                                                      dispatch={[Function]}
                                                      forceRender={false}
                                                      label={
                                                        NamedNode {
                                                          "sI": 59,
                                                          "term": "name",
                                                          "termType": "NamedNode",
                                                          "value": "http://schema.org/name",
                                                        }
                                                      }
                                                      limit={1}
                                                      subject={
                                                        NamedNode {
                                                          "sI": 69,
                                                          "term": "1",
                                                          "termType": "NamedNode",
                                                          "value": "https://argu.dev/o/1",
                                                        }
                                                      }
                                                      version="test1"
                                                    >
                                                      Gemeente Utrecht
                                                    </Property>
                                                  </ConnectedProp>
                                                </linkedSubject[ConnectedProp]>
                                              </a>
                                            </Link>
                                          </LDLink>
                                        </linkedSubject[LDLink]>
                                        <Button
                                          active={false}
                                          alt="Menu uitvouwen of inklappen"
                                          className="SideBarCollapsible__toggle"
                                          data-test="SideBarCollapsible-toggle"
                                          narrow={false}
                                          onClick={[Function]}
                                          plain={true}
                                          small={false}
                                          theme="default"
                                          type="button"
                                        >
                                          <button
                                            alt="Menu uitvouwen of inklappen"
                                            className="Button Button--plain Button--default SideBarCollapsible__toggle"
                                            data-test="Button-button"
                                            onClick={[Function]}
                                            type="button"
                                          >
                                            <span
                                              className="Button__label"
                                            >
                                              <FontAwesome
                                                name="caret-right"
                                              >
                                                <span
                                                  aria-hidden={true}
                                                  className="fa fa-caret-right"
                                                />
                                              </FontAwesome>
                                            </span>
                                          </button>
                                        </Button>
                                      </div>
                                      <Connect(CollapsibleContainer)
                                        group="Navbar"
                                        id="<https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1>-menu-items"
                                      >
                                        <CollapsibleContainer
                                          group="Navbar"
                                          id="<https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1>-menu-items"
                                          onClickToggle={[Function]}
                                          onInitializeCollapsible={[Function]}
                                          opened={false}
                                          startOpened={false}
                                        >
                                          <Collapsible
                                            alwaysMountChildren={true}
                                            group="Navbar"
                                            id="<https://argu.dev/o/find?iri=https%3A%2F%2Fargu.dev%2Fo%2F1>-menu-items"
                                            onClickToggle={[Function]}
                                            onInitializeCollapsible={[Function]}
                                            opened={false}
                                            startOpened={false}
                                          >
                                            <div
                                              aria-expanded={false}
                                              className="Collapsible"
                                            >
                                              <div
                                                className="Collapsible__visible-content"
                                              />
                                              <div
                                                aria-hidden={-1}
                                                className="Collapsible__invisible-content"
                                              >
                                                <ReactCollapse
                                                  isOpened={false}
                                                  springConfig={
                                                    Object {
                                                      "damping": 30,
                                                      "stiffness": 300,
                                                    }
                                                  }
                                                  style={
                                                    Object {
                                                      "minHeight": "undefinedpx",
                                                    }
                                                  }
                                                >
                                                  <div>
                                                    <Connect(LinkedResourceContainer)
                                                      subject={
                                                        NamedNode {
                                                          "sI": 120,
                                                          "term": "menus/organizations",
                                                          "termType": "NamedNode",
                                                          "value": "https://argu.dev/menus/organizations",
                                                        }
                                                      }
                                                    >
                                                      <LinkedResourceContainer
                                                        forceRender={false}
                                                        loadLinkedObject={[Function]}
                                                        reloadLinkedObject={[Function]}
                                                        subject={
                                                          NamedNode {
                                                            "sI": 120,
                                                            "term": "menus/organizations",
                                                            "termType": "NamedNode",
                                                            "value": "https://argu.dev/menus/organizations",
                                                          }
                                                        }
                                                        version="new"
                                                      />
                                                    </Connect(LinkedResourceContainer)>
                                                  </div>
                                                </ReactCollapse>
                                              </div>
                                            </div>
                                          </Collapsible>
                                        </CollapsibleContainer>
                                      </Connect(CollapsibleContainer)>
                                    </div>
                                  </SideBarCollapsible>
                                </Connect(SideBarCollapsible)>
                              </div>
                              <linkedSubject[Connect(Type)]>
                                <Connect(Type)
                                  subject={
                                    NamedNode {
                                      "sI": 69,
                                      "term": "1",
                                      "termType": "NamedNode",
                                      "value": "https://argu.dev/o/1",
                                    }
                                  }
                                >
                                  <Type
                                    dispatch={[Function]}
                                    subject={
                                      NamedNode {
                                        "sI": 69,
                                        "term": "1",
                                        "termType": "NamedNode",
                                        "value": "https://argu.dev/o/1",
                                      }
                                    }
                                    version="test1"
                                  >
                                    <div
                                      className="no-view"
                                    >
                                      <linkedSubject[ConnectedProp]
                                        label={
                                          NamedNode {
                                            "sI": 59,
                                            "term": "name",
                                            "termType": "NamedNode",
                                            "value": "http://schema.org/name",
                                          }
                                        }
                                      >
                                        <ConnectedProp
                                          label={
                                            NamedNode {
                                              "sI": 59,
                                              "term": "name",
                                              "termType": "NamedNode",
                                              "value": "http://schema.org/name",
                                            }
                                          }
                                          subject={
                                            NamedNode {
                                              "sI": 69,
                                              "term": "1",
                                              "termType": "NamedNode",
                                              "value": "https://argu.dev/o/1",
                                            }
                                          }
                                        >
                                          <Property
                                            dispatch={[Function]}
                                            forceRender={false}
                                            label={
                                              NamedNode {
                                                "sI": 59,
                                                "term": "name",
                                                "termType": "NamedNode",
                                                "value": "http://schema.org/name",
                                              }
                                            }
                                            limit={1}
                                            subject={
                                              NamedNode {
                                                "sI": 69,
                                                "term": "1",
                                                "termType": "NamedNode",
                                                "value": "https://argu.dev/o/1",
                                              }
                                            }
                                            version="test1"
                                          >
                                            Gemeente Utrecht
                                          </Property>
                                        </ConnectedProp>
                                      </linkedSubject[ConnectedProp]>
                                      <p>
                                        We currently don"t have a view for this (&lt;https://argu.dev/o/1&gt;)
                                      </p>
                                    </div>
                                  </Type>
                                </Connect(Type)>
                              </linkedSubject[Connect(Type)]>
                              <linkedSubject[ConnectedProp]
                                label={
                                  NamedNode {
                                    "sI": 121,
                                    "term": "baseColor",
                                    "termType": "NamedNode",
                                    "value": "https://argu.co/ns/core#baseColor",
                                  }
                                }
                              >
                                <ConnectedProp
                                  label={
                                    NamedNode {
                                      "sI": 121,
                                      "term": "baseColor",
                                      "termType": "NamedNode",
                                      "value": "https://argu.co/ns/core#baseColor",
                                    }
                                  }
                                  subject={
                                    NamedNode {
                                      "sI": 69,
                                      "term": "1",
                                      "termType": "NamedNode",
                                      "value": "https://argu.dev/o/1",
                                    }
                                  }
                                >
                                  <Property
                                    dispatch={[Function]}
                                    forceRender={false}
                                    label={
                                      NamedNode {
                                        "sI": 121,
                                        "term": "baseColor",
                                        "termType": "NamedNode",
                                        "value": "https://argu.co/ns/core#baseColor",
                                      }
                                    }
                                    limit={1}
                                    subject={
                                      NamedNode {
                                        "sI": 69,
                                        "term": "1",
                                        "termType": "NamedNode",
                                        "value": "https://argu.dev/o/1",
                                      }
                                    }
                                    version="test1"
                                  />
                                </ConnectedProp>
                              </linkedSubject[ConnectedProp]>
                            </div>
                          </LinkedResourceContainer>
                        </Connect(LinkedResourceContainer)>
                      </Contains>
                    </Connect(Contains)>
                  </linkedSubject[Connect(Contains)]>
                </Property>
              </ConnectedProp>
            </linkedSubject[ConnectedProp]>
          </div>
        </LinkedResourceContainer>
      </Connect(LinkedResourceContainer)>
    </div>
    <div
      className="NavBarContent__footer"
    >
      <Connect(LinkedResourceContainer)
        subject={
          NamedNode {
            "sI": 117,
            "term": "c_a",
            "termType": "NamedNode",
            "value": "https://argu.dev/c_a",
          }
        }
        topology={
          NamedNode {
            "sI": 44,
            "term": "sidebar",
            "termType": "NamedNode",
            "value": "https://argu.co/ns/core#sidebar",
          }
        }
      >
        <LinkedResourceContainer
          forceRender={false}
          loadLinkedObject={[Function]}
          reloadLinkedObject={[Function]}
          subject={
            NamedNode {
              "sI": 117,
              "term": "c_a",
              "termType": "NamedNode",
              "value": "https://argu.dev/c_a",
            }
          }
          topology={
            NamedNode {
              "sI": 44,
              "term": "sidebar",
              "termType": "NamedNode",
              "value": "https://argu.co/ns/core#sidebar",
            }
          }
          version="new"
        />
      </Connect(LinkedResourceContainer)>
      <Connect(LinkedResourceContainer)
        subject={
          NamedNode {
            "sI": 118,
            "term": "n?type=infinite",
            "termType": "NamedNode",
            "value": "https://argu.dev/n?type=infinite",
          }
        }
        topology={
          NamedNode {
            "sI": 44,
            "term": "sidebar",
            "termType": "NamedNode",
            "value": "https://argu.co/ns/core#sidebar",
          }
        }
      >
        <LinkedResourceContainer
          forceRender={false}
          loadLinkedObject={[Function]}
          reloadLinkedObject={[Function]}
          subject={
            NamedNode {
              "sI": 118,
              "term": "n?type=infinite",
              "termType": "NamedNode",
              "value": "https://argu.dev/n?type=infinite",
            }
          }
          topology={
            NamedNode {
              "sI": 44,
              "term": "sidebar",
              "termType": "NamedNode",
              "value": "https://argu.co/ns/core#sidebar",
            }
          }
          version="new"
        />
      </Connect(LinkedResourceContainer)>
      <Connect(LinkedResourceContainer)
        subject={
          NamedNode {
            "sI": 119,
            "term": "menus/info",
            "termType": "NamedNode",
            "value": "https://argu.dev/menus/info",
          }
        }
        topology={
          NamedNode {
            "sI": 44,
            "term": "sidebar",
            "termType": "NamedNode",
            "value": "https://argu.co/ns/core#sidebar",
          }
        }
      >
        <LinkedResourceContainer
          forceRender={false}
          loadLinkedObject={[Function]}
          reloadLinkedObject={[Function]}
          subject={
            NamedNode {
              "sI": 119,
              "term": "menus/info",
              "termType": "NamedNode",
              "value": "https://argu.dev/menus/info",
            }
          }
          topology={
            NamedNode {
              "sI": 44,
              "term": "sidebar",
              "termType": "NamedNode",
              "value": "https://argu.co/ns/core#sidebar",
            }
          }
          version="new"
        />
      </Connect(LinkedResourceContainer)>
      <div
        className="NavBarContent__center-footer"
      >
        <button
          onClick={[Function]}
          style={
            Object {
              "bottom": "0em",
              "color": "white",
              "cursor": "pointer",
              "left": "0em",
              "padding": "1em",
              "position": "absolute",
            }
          }
          title="Back to classic Argu"
        >
          <span
            className="fa fa-close"
          />
        </button>
        <Link
          className="NavBarContent__logo"
          replace={false}
          to="/"
        >
          <a
            className="NavBarContent__logo"
            href="/"
            onClick={[Function]}
          >
            <svg
              height="28"
              viewBox="0 0 211 108"
              width="37"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                height="8.3"
                width="51.2"
                x="159.8"
                y="86.4"
              />
              <rect
                height="8.4"
                width="86.9"
                y="86.2"
              />
              <rect
                height="8.4"
                width="211"
                y="14.3"
              />
              <path
                d="M20.3 81c-2.5 0-4.8-0.4-6.9-1.2 -2.1-0.8-4-1.9-5.5-3.4 -1.5-1.4-2.7-3.1-3.6-5.1 -0.9-2-1.3-4.1-1.3-6.4 0-2.4 0.5-4.6 1.6-6.6s2.5-3.7 4.5-5.1c1.9-1.4 4.2-2.5 6.9-3.3 2.6-0.8 5.6-1.2 8.8-1.2 2.4 0 4.8 0.2 7.2 0.6 2.4 0.4 4.5 1 6.4 1.7v-3.5c0-3.8-1.1-6.7-3.2-8.9 -2.2-2.1-5.3-3.2-9.3-3.2 -2.7 0-5.4 0.5-8.1 1.5 -2.6 1-5.4 2.5-8.2 4.4l-3.3-6.9c6.6-4.4 13.5-6.7 20.6-6.7 7.1 0 12.5 1.8 16.5 5.4 3.9 3.6 5.9 8.8 5.9 15.5v18.5c0 1.2 0.2 2.1 0.7 2.6 0.5 0.5 1.2 0.8 2.4 0.9v9.2c-1 0.2-2 0.3-2.9 0.4 -0.9 0.1-1.7 0.1-2.4 0.1 -2.1-0.1-3.7-0.7-4.8-1.7s-1.7-2.3-2-3.9l-0.3-3.2c-2.3 3-5.1 5.3-8.5 6.9C27.7 80.2 24.1 81 20.3 81zM23.2 73.1c2.6 0 5.1-0.5 7.4-1.4 2.3-0.9 4.1-2.3 5.4-4 1.4-1.2 2.1-2.4 2.1-3.7v-6.8c-1.8-0.7-3.8-1.3-5.8-1.7 -2.1-0.4-4.1-0.6-6-0.6 -3.9 0-7 0.8-9.5 2.5 -2.4 1.7-3.7 3.9-3.7 6.6 0 2.5 0.9 4.7 2.8 6.4C17.9 72.2 20.3 73.1 23.2 73.1z"
              />
              <path
                d="M90.8 38.1c-4.2 0.1-7.9 1.1-11.1 2.9 -3.2 1.8-5.5 4.4-7 7.7v31.4H62V28.8h10v11.5c1.8-3.5 4.2-6.4 7-8.6 2.8-2.2 5.9-3.3 9.1-3.3 0.7 0 1.2 0 1.6 0 0.4 0 0.8 0 1.1 0.1V38.1z"
              />
              <path
                d="M119.1 80.5c-3.5 0-6.6-0.7-9.5-2.1 -2.8-1.4-5.3-3.3-7.4-5.7 -2.1-2.4-3.7-5.2-4.8-8.3 -1.1-3.1-1.7-6.4-1.7-9.9 0-3.7 0.6-7.1 1.7-10.3 1.1-3.2 2.8-6 4.9-8.4 2.1-2.4 4.6-4.3 7.5-5.7 2.9-1.4 6.1-2.1 9.7-2.1 4.2 0 7.8 1 10.9 2.9 3.1 2 5.6 4.5 7.6 7.6v-9.7h9.5v49.8c0 3.7-0.7 7-2.1 9.9 -1.4 2.9-3.3 5.3-5.7 7.3 -2.4 2-5.3 3.5-8.7 4.6 -3.3 1-6.9 1.6-10.8 1.6 -5.7 0-10.4-1-14.1-2.9 -3.7-1.9-6.9-4.6-9.5-8.1l6.1-5.7c2 2.7 4.5 4.8 7.6 6.3 3.1 1.4 6.4 2.2 9.9 2.2 2.2 0 4.3-0.3 6.3-0.9 2-0.6 3.7-1.5 5.2-2.7 1.5-1.2 2.7-2.8 3.6-4.7 0.9-1.9 1.3-4.1 1.3-6.8v-7.8c-1.8 3.1-4.3 5.6-7.5 7.3C125.9 79.7 122.6 80.5 119.1 80.5zM122.6 71.8c1.6 0 3.2-0.3 4.7-0.8 1.5-0.6 2.9-1.3 4.2-2.3 1.3-0.9 2.4-2 3.3-3.2 0.9-1.2 1.6-2.5 2-3.9V48.2c-1.3-3.3-3.3-5.9-6.1-8 -2.7-2.1-5.7-3.1-8.8-3.1 -2.4 0-4.5 0.5-6.3 1.5 -1.9 1-3.4 2.4-4.8 4 -1.3 1.7-2.3 3.5-3 5.6 -0.7 2.1-1.1 4.2-1.1 6.5 0 2.4 0.4 4.6 1.2 6.6 0.8 2.1 1.9 3.9 3.4 5.4 1.4 1.5 3.1 2.8 5 3.7C118.2 71.4 120.3 71.8 122.6 71.8z"
              />
              <path
                d="M175.9 81c-5.3 0-9.3-1.7-12.1-5.2 -2.7-3.5-4.1-8.7-4.1-15.6v-31.4h10.8V58c0 9.2 3.2 13.8 9.7 13.8 3.1 0 6.1-0.9 8.8-2.8 2.7-1.9 4.9-4.6 6.4-8.2V28.8h10.8v38.5c0 1.2 0.2 2.1 0.7 2.6 0.5 0.5 1.3 0.8 2.5 0.9v9.2c-1.2 0.2-2.2 0.3-3 0.4 -0.8 0.1-1.6 0.1-2.3 0.1 -4.1-0.3-6.3-2.2-6.9-5.6l-0.2-5.3c-2.3 3.7-5.3 6.6-8.9 8.5C184.4 80.1 180.3 81 175.9 81z"
              />
            </svg>
          </a>
        </Link>
      </div>
    </div>
  </div>
</NavBarContent>
`;
